app/helpers/application_helper.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  include FormBuilderHelper
```

Comment:
~have a look at the `form_builder.rb` class in the provider app as prefer that model of adding this helper. :)~

This shouldn't be needed (I don;t think)